### ISSUE TITLE
The VoCo pre-training loss in nnssl does not decrease.

### ISSUE THREAD

**Issue by Lry777** [MIC-DKFZ/nnssl#22]
The pre-training strategy in VoCoTrainer is inconsistent with the VoCo source code. In the VoCo source code, VoCoHead includes gradient momentum control for both teacher and student branches (similar to MoCo). The missing part in VoCoTrainer's loss calculation is likely the main reason why the loss doesn't decrease.

After adding momentum control to VoCoTrainer, the loss decreased normally!
